src/core/libraries/gnmdriver/gnmdriver.cpp

```diff
@@ -29,7 +29,7 @@ namespace Libraries::GnmDriver {
 
 using namespace AmdGpu;
 
-enum GnmEventIdents : u64 {
+enum GnmEqEventType : u64 {
```

Comment:
Is the name change necessary?

Reply:
I would say so, this is so far not used and more inline with sceGnmGetEqEventType.